environment.py

```diff
@@ -362,11 +364,24 @@ def env() -> Mapping[str, Optional[str]]:
         # Typically only set for main deployments.
         'AZUL_ENABLE_MONITORING': '0',
 
-        # The default bundle UUID prefix to use for reindexing bundles in the DSS
-        # and for subscriptions to the DSS. If this variable is set to a non-empty
-        # string, only bundles whose UUID starts with the specified string will be
-        # indexed.
-        'AZUL_DSS_QUERY_PREFIX': '',
+        # Identifies the DSS repository endpoint and prefix to index.
+        # The syntax in EBNF is:
+        #
+        # source = 'https://', net_loc,
```

Comment:
Still don't understand why we need to pick apart the URL. Just summarize it as `endpoint` and informally define `endpoint` to be the URL of the DSS instance.